src/test/isolation2/sql/gdd/prepare.sql

```
@@ -49,17 +47,13 @@ SELECT segid(0,10) is not null;
 SELECT segid(1,10) is not null;
 SELECT segid(2,10) is not null;
 
--- table to just store the master's data directory path on segment.
-CREATE TABLE datadir(a int, dir text);
-INSERT INTO datadir select 1,datadir from gp_segment_configuration where role='p' and content=-1;
-
-ALTER SYSTEM SET gp_enable_global_deadlock_detector TO on;
-ALTER SYSTEM SET gp_global_deadlock_detector_period TO 5;
+--enable GDD
+-- start_ignore
+! gpconfig -c gp_enable_global_deadlock_detector -v on;
+! gpconfig -c gp_global_deadlock_detector_period -v 5;
+! gpstop -rai;
+-- end_ignore
```

I understand that we always do this in the tests, but we also know that if there are errors in the ignore block they're silently ignored, which can cause unexpected failures in later tests and is uneasy to debug.

Please consider using `matchsubs` instead of `ignore`.